storybook-static-preview/index.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Deploy preview</title>
+    <link rel="stylesheet" type="text/css" href="https://unpkg.com/carbon-components@latest/css/carbon-components.min.css">
+    <style type="text/css">
```

Comment:
tiny nitpick but should we center these buttons on the page?

Reply:
OK makes sense - Done.